src/v2/Components/__stories__/Molecules.story.tsx

```diff
@@ -476,3 +483,629 @@ export const Content = () => {
     </Page>
   )
 }
+
+const SwiperWithProgress: React.FC<
+  SwiperProps & { verticalAlign?: "top" | "bottom" }
+> = ({ children, verticalAlign = "bottom", ...rest }) => {
+  const count = React.Children.count(children)
+  const [boxProps, swiperProps] = splitBoxProps(rest)
```

Comment:
What does the splitBoxProps do?

Reply:
Breaks the rest props into boxProps and not(boxProps)... it's useful if you want the outer container to be positionable using box props but still want to pass things to the underlying component (in this case the swiper). Avoids having to whitelist everything. https://github.com/artsy/palette/blob/master/packages/palette/src/utils/splitProps.ts

Reply:
Ahh i see, nice 👍